services/search/pkg/query/bleve/compiler.go

```diff
@@ -264,3 +287,76 @@ func normalizeGroupingProperty(group *ast.GroupNode) *ast.GroupNode {
 	}
 	return group
 }
+
+func mimeType(k, v string) (bleveQuery.Query, bool) {
+	switch v {
+	case "file":
+		q := bleve.NewBooleanQuery()
+		q.AddMustNot(bleveQuery.NewQueryStringQuery(k + ":httpd/unix-directory"))
+		return q, false
+	case "folder":
+		return bleveQuery.NewQueryStringQuery(k + ":httpd/unix-directory"), false
+	case "documents":
```

Comment:
i know in the user story/issue there is also plural used but I would vote that we do the singular `document`, `spreadsheet` etc. Found it counterintuitive to use plural, what do you think?

Reply:
i mean `file`, `folder` is also singular

Reply:
It sounds to me like the keywords should be singular. If we take a look at google drive we can see that the filter `titles` and `keywords` are different `https://drive.google.com/drive/my-drive?q=type:document`
@fschade Does it make sense to improve the capabilities and add some filters `title` that Users see on UI bun in a request to use corresponding `keywords`? 
<img width="194" alt="Screenshot 2023-11-07 at 09 56 23" src="https://github.com/owncloud/ocis/assets/11679989/905c7b4e-5cee-4c16-a9d5-f8fec5d8ee5c">

Reply:
yes you are right, i think the "alias" or "keyword" is for us to decide, for how its displayed in the UI i will ask @tbsbdr  because he knows it best :)

Reply:
i mean if we put it in a sentence `I would like to search for file with a title 'fileXY' and of the type 'Document'` sounds best

Reply:
@2403905 talked to @tbsbdr and he prefers singular in the UI, so lets keep it consistent with the "alias"/keywords if thats fine with you :)

Reply:
@2403905 can you explain it in more detail? I don not get the question